eZ/Publish/Core/Repository/SiteAccessAware/RoleService.php

```diff
@@ -0,0 +1,215 @@
+<?php
+
+/**
+ * RoleService class.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\Repository\SiteAccessAware;
+
+use eZ\Publish\API\Repository\RoleService as RoleServiceInterface;
+use eZ\Publish\API\Repository\Values\User\Limitation\RoleLimitation;
+use eZ\Publish\API\Repository\Values\User\Policy;
+use eZ\Publish\API\Repository\Values\User\PolicyCreateStruct;
+use eZ\Publish\API\Repository\Values\User\PolicyDraft;
+use eZ\Publish\API\Repository\Values\User\PolicyUpdateStruct;
+use eZ\Publish\API\Repository\Values\User\Role;
+use eZ\Publish\API\Repository\Values\User\RoleAssignment;
+use eZ\Publish\API\Repository\Values\User\RoleCreateStruct;
+use eZ\Publish\API\Repository\Values\User\RoleDraft;
+use eZ\Publish\API\Repository\Values\User\RoleUpdateStruct;
+use eZ\Publish\API\Repository\Values\User\User;
+use eZ\Publish\API\Repository\Values\User\UserGroup;
+
+/**
+ * RoleService for SiteAccessAware layer.
+ *
+ * Currently does nothing but hand over calls to aggregated service.
+ */
+class RoleService implements RoleServiceInterface
```

Comment:
Agreed, can drop it, only reason I kept it in is that this will most likely be first service we will need to add prioritized languages on top of those that have it already :)